packages/trader/src/Modules/Contract/Containers/replay-chart.tsx

```diff
@@ -47,7 +47,7 @@ const ReplayChart = observer(
         const from_platform = getPlatformRedirect(app_routing_history);
         const should_force_light_theme = from_platform.name === 'DBot';
         const settings = {
-            language: current_language.toLowerCase(),
+            language: current_language,
```

Comment:
After this PR release in SmartCharts https://github.com/deriv-com/SmartCharts/pull/1605, language value case will be handled by the chart.